components/rfid-reader/rdm6300_serial/rdm6300_serial.py

```diff
@@ -0,0 +1,142 @@
+"""
+The RDM6300 / RDM630 connected via serial UART port
+
+The Rdm6300Reader supports 3 number formats for the card ID. See NUMBER_FORMAT below for details.
+
+"""
+import os
+import logging
+import configparser
+import serial
+
+import base.inputminus as pyil
+from base.simplecolors import colors
+from base.reader_base import *
+
+from .description import DESCRIPTION
+
+
+# Create logger
+logger = logging.getLogger(os.path.basename(__file__).ljust(25))
+logger.setLevel(logging.DEBUG)
+# Create console handler and set default level (may be overwritten by readersupport module)
+logconsole = logging.StreamHandler()
+logconsole.setFormatter(logging.Formatter('%(asctime)s - %(name)s - %(levelname)-8s: %(message)s', datefmt='%d.%m.%Y %H:%M:%S'))
+logconsole.setLevel(logging.INFO)
+logger.addHandler(logconsole)
+
+NUMBER_FORMAT = ['card_id_dec',
+                 'card_id_raw',
+                 'card_id_float']
+NUMBER_FORMAT_HELP = ['A 10 digit card ID e.g. 0006762840 (default)',
```

Comment:
The initial Idea was to keep the Number format raw as default, that people who will update will not break their card assignments. 
From technical perceptive, there is no point in keeping the RAW format, its just for backward compatibility.

Reply:
And that is what I intended: making the default case the backward compatible case. I got it mixed up, obviously. I will look into it and fix it. 

Happy you like it :-)